server.js

```diff
@@ -20,9 +20,9 @@ const serverConfig = {
   // case of the profiler, which is a quite heavy program.
   hot: false,
   liveReload: false,
-  contentBase: config.output.path,
-  publicPath: config.output.publicPath,
```

Comment:
This was moved to other configuration, but just removing them seemed to work on my side too.

Reply:
I think this was needed before we used the copy-webpack-plugin to copy things (we were using a `cp` to the dist directory before).

Reply:
I added `static: false` at the end, otherwise the server would expose a (non existent) /public directory.